models/Store.js

```diff
@@ -4,7 +4,7 @@ const mongoose = require('mongoose');
 
 const schema = new mongoose.Schema({
   name: { type: String, required: true },
-  address: { type: String, required: true },
+  address: { type: Object, required: true },
```

Comment:
Is the reason that you're using a `Object` as the type rather than specifying the structure because you're worried that yelp might change it's own schema?

I'm not sure what you're using the address field for on the front end, but from what I saw you're only using nested display_address here, which would just be ` { type: [String ], required: true }`.

Reply:
there's also address1, but I guess that's the same as display_address[0] practically. I use address1, address2, city, state, and zip_code is displaying results on StoreResult

Reply:
Ah. Yes. Forgot about that view. Ignore this.